builtin/logical/jwt/path_issue.go

```diff
@@ -0,0 +1,133 @@
+package jwt
+
+import (
+	"fmt"
+	"time"
+	"encoding/json"
+
+	"github.com/hashicorp/vault/helper/uuid"
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathIssue(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: `issue/(?P<role>\w[\w-]+\w)`,
+		Fields: map[string]*framework.FieldSchema{
+			"role": &framework.FieldSchema{
+				Type: framework.TypeString,
+				Description: "The desired role with configuration for this request",
+			},
+			"iss": &framework.FieldSchema{
```

Comment:
We should be explicit with these fields. "iss" should be "issuer", "sub" should be "subject" etc. I'd prefer clarity over saving a few bytes.